src/plugins/intel_gpu/include/intel_gpu/runtime/debug_configuration.hpp

```diff
@@ -113,6 +113,7 @@ class debug_configuration {
     int dump_layers_dst_only;                                   // Dump only output of layers
     int dump_layers_result;                                     // Dump result layers
     int dump_layers_input;                                      // Dump input layers
+    int print_layers_input;                                     // Print the input data_shape for benchmark_app. This option is related to dump_iteration.
```

Comment:
* please move it to different place
* What about "print_input_data_shapes" instead

Reply:
updated PR